From e10ceebf2487b344b5ccdc7d8647a430c2b311af Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?S=C3=B6ren=20Tempel?= <soeren+git@soeren-tempel.net>
Date: Mon, 15 Mar 2021 22:37:39 +0100
Subject: [PATCH] Add missing <limits> include

Needed for the definition of the std::numeric_limits type.
---
 zip_archive_stream_entry.cc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/zip_archive_stream_entry.cc b/zip_archive_stream_entry.cc
index 248d316..fb1405e 100644
--- a/zip_archive_stream_entry.cc
+++ b/zip_archive_stream_entry.cc
@@ -23,6 +23,7 @@
 #include <sys/types.h>
 #include <unistd.h>
 
+#include <limits>
 #include <memory>
 #include <vector>
 
